validation-test/SIL/verify_all_overlays.sil

```diff
@@ -1,3 +1,3 @@
-// RUN: for x in %platform-sdk-overlay-dir/*.swiftmodule; do [[ $(basename "$x") = Swift.swiftmodule ]] && continue; llvm-bcanalyzer $x | FileCheck %s; %target-sil-opt -enable-sil-verify-all $x > /dev/null; done
+// RUN: for x in %platform-sdk-overlay-dir/*.swiftmodule; do [[ $(basename "$x") = Swift.swiftmodule ]] && continue; llvm-bcanalyzer $x | FileCheck %s; %target-sil-opt -sdk %sdk -enable-sil-verify-all $x > /dev/null; done
```

Comment:
That being said, `%target-sil-opt` is configured with the correct `-target` and resource dir on each platform. I think `-sdk` isn't specified in order to run tests like `test/SIL/Parser/unnamed_struct_identifiers.sil`:

```
// RUN: %target-sil-opt -sdk %clang-importer-sdk-path %s
```

Adding `-sdk` to the `%target-sil-opt` substitution would mean writing the above test a little differently. For now, I think adding the `-sdk` here is the path of least resistance. :)